docs/Features/XMLColorsFeatures.md

```diff
@@ -0,0 +1,70 @@
+# XML Colors Features
+
+XML colors support provides the capability to mark a DOM node (attribute or text) as color with the `xml.colors` settings by using XPath expression : 
+
+ * `color/text()` defines the text node of the `color` element.
+ * `item/@color` defines the `color` attribute node of the `item` element.
+
+## Text node color (color/text())
```

Comment:
Consider changing the title to something like (maybe for attribute value as well):
Define Color in Text Content with `color/text()`

Reply:
fixed